### DISCUSSION TITLE
Who is on the team?

### DISCUSSION THREAD

**Discussion by MilesWK** [EnZon3/Drop#17] [General]
Hello! I wanted to know who all is working on this project and if you would like help. I love CSS and HTML and could help in that field.

**Comment by EnZon3** [comment]
> Hello! I wanted to know who all is working on this project and if you would like help. I love CSS and HTML and could help in that field. 

Hey Miles! As of now I am the only core maintainer of the project, however feel absolutely free to open a PR if you would like to contribute/add anything!

**Reply by MilesWK** [reply]
That's okay. I was just wondering! Thanks for answering
